stock_picking_partner_note/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    check_note_already_in_use = fields.Boolean(
+        string="Check picking note already in use",
```

Comment:
@santostelmo You should explain in which flows (update / delete)

Reply:
Added some help and a line in readme